net/lp_rpc.proto

```diff
@@ -211,16 +214,21 @@ message NotifySegment {
     // URL of the segment to transcode.
     string url      = 1;
 
-    // Job the segment belongs to.
-    string job      = 2;
+    // Configuration for the transcoding job
+    SegData segData = 3;
 
     // ID for this particular transcoding task.
     int64 taskId   = 16;
 
-    // Set of profiles to transcode this segment into.
+// Deprecated fields. May still be populated sometimes for back compat
+
+    // Deprecated by segData. Job the segment belongs to.
+    string job      = 2;
```

Comment:
recommended way of deprecating field that's not used anymore
```suggestion
    // string job      = 2;
    reserved 2;
    reserved "job";
```

Reply:
Good idea, reserved the number in 4d957856cdf5136a1e21972169e9f50d64e244c4 . Didn't reserve the field name since we don't do JSON, so there should not be a conflict. (Can still add that in if you feel it should be there though.)